src/main/java/mil/dds/anet/resources/AttachmentResource.java

```diff
@@ -220,6 +221,46 @@ public Response viewAttachment(final @Auth Person user, @PathParam("uuid") Strin
     return response.build();
   }
 
+  /**
+   * Gets an image attachment cropped by the given coordinates
+   * 
+   * @param user the user calling the operation
+   * @param attachmentUuid the attachment uid
+   * @param left crop left coordinate
+   * @param top crop top coordinate
+   * @param width crop width coordinate
+   * @param height crop height coordinate
+   * @return the cropped image
+   */
+  @GET
+  @Path("/view-cropped/{attachmentUuid}/{left}/{top}/{width}/{height}")
+  public Response viewAttachmentCropped(final @Auth Person user,
+      @PathParam("attachmentUuid") String attachmentUuid, @PathParam("left") int left,
+      @PathParam("top") int top, @PathParam("width") int width, @PathParam("height") int height
+
+  ) {
+    assertAttachmentEnabled();
+    try {
+      final Attachment attachment = getAttachment(attachmentUuid);
+      ByteArrayOutputStream output = new ByteArrayOutputStream();
+      dao.streamContentBlob(attachmentUuid, output);
+      BufferedImage image = ImageIO.read(new ByteArrayInputStream(output.toByteArray()));
```

Comment:
This loads the entire attachment into memory, which we avoid everywhere else. Is there no other way?

Reply:
Addressed by cropping now in front-end